IntegrationTests/tests_04_performance/test_01_resources/shared.swift

```diff
@@ -136,3 +136,119 @@ func withAutoReleasePool<T>(_ execute: () throws -> T) rethrows -> T {
     }
     #endif
 }
+
+// MARK: UDP Shared
+
+/// Repeatedly send, echo and receive a number of UDP requests.
+/// Based on the UDPEchoClient/Server example.
+enum UdpShared {
```

Comment:
nit: in Swift, abbreviations go all caps (or all lowercase, depending on where they are). This would be `UDPShared`

Reply:
Ok, will go capitals.

Reply:
Changed.